snyk-monitor/templates/pvc.yaml

```diff
@@ -13,5 +13,11 @@ spec:
   resources:
     requests:
       storage: {{ .Values.temporaryStorageSize }}
-  storageClassName: {{ .Values.pvc.storageClassName }}
+  {{- if .Values.pvc.storageClassName }}
+  {{- if (eq "-" .Values.pvc.storageClassName) }}
+  storageClassName: ""
+  {{- else }}
+  storageClassName: "{{ .Values.pvc.storageClassName }}"
+  {{- end }}
+{{- end }}
```

Comment:
Should indentation match the first `{{-if }}` case?

Reply:
fixed indentation and commit message